frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget/ParameterFieldWidget.jsx

```diff
@@ -26,7 +26,7 @@ const propTypes = {
   parameters: PropTypes.array.isRequired,
   placeholder: PropTypes.string.isRequired,
   setValue: PropTypes.func.isRequired,
-  value: PropTypes.string,
+  value: PropTypes.arrayOf(PropTypes.string),
```

Comment:
:x: 

It looks like this component is meant to work both with strings and arrays, see [this line](https://github.com/metabase/metabase/blob/2146c2ada057bdfbf68f7bbde978dc44994d2b69/frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget/ParameterFieldWidget.jsx#L50C3-L50C76).

```suggestion
  value: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.arrayOf(PropTypes.string),
  ]),
```

Reply:
I haven't seen a scenario in practice in which it's ever a string, but you are absolutely right!
This is a safer option.

Thanks for being so vigilant.